elastic/apm-server
# Add format to transaction.duration.us

## Motivation/summary
Adds input- and output-format and precision to `transaction.duration.us` for improved usability and 

What's controversal here is that we format a field that has `us` in its name to `milliseconds`. 

## Checklist

~- [ ] I have signed the [Contributor License Agreement](https://www.elastic.co/contributor-agreement/).~

~- [ ] I have made corresponding changes to the documentation~
~- [ ] I have added tests that prove my fix is effective or that my feature works~

- [ ] needs update in Kibana

## How to test these changes
1. Check out a local copy of Kibana and update the index pattern to contain the new format; start Kibana and add APM data 
2. navigate through APM UI and check out implications on Kibana data for `transaction.duration.us`. 




## Related issues
implements #3751
